File: tree-hugger/lib/tests/fixtures/imports.java
// Test fixture for Java import extraction
package com.example.test;

// Simple import
import java.util.List;

// Import with wildcard (captured as single import)
import java.io.*;

// Static import
import static java.lang.Math.PI;

// Multiple imports from same package
import java.util.HashMap;
import java.util.ArrayList;

public class ImportsTest {
    public static void main(String[] args) {
        List<String> list = new ArrayList<>();
        HashMap<String, String> map = new HashMap<>();
        System.out.println(PI);
    }
}

File: tree-hugger/lib/tests/fixtures/imports.scala
// Test fixture for Scala import extraction
package com.example.test

// Simple import
import scala.collection.mutable.ListBuffer

// Wildcard import
import java.util._

// Multiple imports
import scala.io.{Source, StdIn}

// Renamed import
import java.util.{List => JList, Map => JMap}

object ImportsTest {
  def main(args: Array[String]): Unit = {
    val buffer = ListBuffer[String]()
    val list: JList[String] = new java.util.ArrayList()
  }
}
